fusor-ember-cli/app/mixins/needs-deployment-mixin.js

```diff
@@ -6,6 +6,7 @@ export default Ember.Mixin.create({
   applicationController: Ember.inject.controller('application'),
 
   isStarted: Ember.computed.alias("deploymentController.isStarted"),
-  isNotStarted: Ember.computed.alias("deploymentController.isNotStarted")
+  isNotStarted: Ember.computed.alias("deploymentController.isNotStarted"),
 
+  disableCancel: Ember.computed.alias("isStarted")
```

Comment:
@jmrodri, since I was using `disableCancel` in the shared templates for Steps 1,2,3 for both new deployments and existing templates, I needed to define `disableCancel` here for existing deployments.